### ISSUE TITLE
Crucible 

### ISSUE THREAD

**Issue by vicobill** [aws/lumberyard#531]
Can you open source Crucible ,like DOOM3 did？You can use GPL license to game code . 

We love lumberyard, but it's too hard to learn, The sample game is so simple. In UE4, we can learn from Unreal Tournament 2004, but in Lumberyard, what is the choice?

**Comment by AMZN-Gene** [comment]
I agree, it would be great to share the inner workings of Crucible.  I'll pass your suggestion to the rest of the team and see what comes up :)

**Comment by vicobill** [comment]
> I agree, it would be great to share the inner workings of Crucible. I'll pass your suggestion to the rest of the team and see what comes up :)

Thank you very very much, cant much more!

**Comment by wcbx** [comment]
Make it a gem :)
